ibis/expr/window.py

```diff
@@ -68,10 +54,11 @@ def _validate_frame(self):
             f_tuple = isinstance(self.following, tuple)
             has_f = True
 
-        if ((p_tuple and has_f) or (f_tuple and has_p)):
-            raise com.IbisInputError('Can only specify one window side '
-                                     ' when you want an off-center '
-                                     'window')
+        if (p_tuple and has_f) or (f_tuple and has_p):
```

Comment:
The naming is hard to understand here.

Reply:
Indeed, I was really just removing the unnecessary parentheses, but I can address the naming as well.